jscomp/bsb/bsb_global_backend.mli

```diff
@@ -22,8 +22,15 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
-val cmdline_backend : Bsb_config_types.compilation_kind_t option ref
+val backend_is_set : bool ref
 
-val backend : Bsb_config_types.compilation_kind_t Lazy.t
+val backend : Bsb_config_types.compilation_kind_t ref
+
+val lib_artifacts_dir : string ref
+
+val lib_ocaml_dir : string ref
+
+val backend_string: string ref
+
+val set_backend : Bsb_config_types.compilation_kind_t -> unit
```

Comment:
can we have some docs for this module, like a comment string for each value

Reply:
Done!